src/js/app/controllers/community/CommunitySettingsCtrl.js

```diff
@@ -5,9 +5,16 @@ var controller = function ($scope, $history, $analytics, community, currentUser,
   _.merge(community, extraProperties);
   var origin = $location.absUrl().replace($location.path(), '');
   $scope.join_url = origin + '/c/' + community.slug + '/join/' + community.beta_access_code;
+  $scope.add_slack_url = origin + '/noo/community/' + community.id + '/settings/slack';
   $scope.community = community;
   $scope.settings = community.settings;
 
+  if ($location.$$search.slack === "1") {
```

Comment:
I discovered it by logging the $location object to see if there was any easy way to get query string params.